dhcpv4/server4/server.go

```diff
@@ -3,24 +3,27 @@ package server4
 import (
 	"log"
 	"net"
-	"sync"
 
 	"github.com/insomniacslk/dhcp/dhcpv4"
 )
 
 /*
   To use the DHCPv4 server code you have to call NewServer with two arguments:
+  - an address to listen on, and
   - a handler function, that will be called every time a valid DHCPv4 packet is
-    received, and
-  - an address to listen on.
+    received.
+
+  The address to listen on is used to know IP address, port and optionally the
+  scope to create and UDP socket to listen on for DHCPv4 traffic.
 
   The handler is a function that takes as input a packet connection, that can be
   used to reply to the client; a peer address, that identifies the client sending
   the request, and the DHCPv4 packet itself. Just implement your custom logic in
   the handler.
 
-  The address to listen on is used to know IP address, port and optionally the
-  scope to create and UDP socket to listen on for DHCPv4 traffic.
+  Optionally, NewServer can receive options that will modify the server object.
```

Comment:
this block just fixes documentation. It's an unnecessary burden to update docs at every API change, so in the future I'll remove example comments and switch to an examples directory